client/jetpack-connect/style.scss

```diff
@@ -1,9 +1,20 @@
+$colophon-height: 50px;
```

Comment:
😢

Reply:
No good? I thought this was appropriate because:

- It's self-contained, meaning this style only applies on this view.
- It's a descriptive name for a variable that is used in a couple of different places here.

Reply:
No no, your reasoning is just fine 👍 Sorry for not clarifying.

My reaction was more about the _need_ for such a parameter. Ideally, we would be flexible and wouldn't need to care about the dimensions of the things in the footer.

But for now I think this is good 👍